node/src/reactor/main_reactor/control.rs

```diff
@@ -52,21 +52,27 @@ impl MainReactor {
         rng: &mut NodeRng,
     ) -> (Duration, Effects<MainEvent>) {
         match self.state {
-            ReactorState::Initialize => match self.initialize_next_component(effect_builder) {
-                Some(effects) => (self.control_logic_default_delay.into(), effects),
-                None => {
-                    if false == self.net.has_sufficient_fully_connected_peers() {
-                        info!("Initialize: awaiting sufficient fully-connected peers");
-                        return (self.control_logic_default_delay.into(), Effects::new());
-                    }
-                    if let Err(msg) = self.refresh_contract_runtime() {
-                        return (Duration::ZERO, fatal!(effect_builder, "{}", msg).ignore());
+            ReactorState::Initialize => {
+                // We can be more greedy when cranking through the initialization process as the
+                // progress is expected to happen quickly.
+                let initialization_logic_default_delay = self.control_logic_default_delay / 4;
```

Comment:
I'd like to have that `4` as a constant, something like `INITIALIZATION_DELAY_SPEED_UP_FACTOR`.

Reply:
Sure, good idea.
I'll address this in a separate PR, since this just got merged.

Reply:
Addressed [here](https://github.com/casper-network/casper-node/pull/3937).